docs/pages/versions/unversioned/sdk/localization.md

```diff
@@ -64,53 +67,87 @@ i18n.locale = Localization.locale;
 
-function App() {
+export default App => {
   return (
-    <Text>
-      {i18n.t('welcome')} {i18n.t('name')}
-    </Text>
+    <View style={styles.container}>
+      <Text style={styles.text}>
+        {i18n.t('welcome')} {i18n.t('name')}
+      </Text>
+    </View>
   );
-}
+};
+
+/* @hide const styles = StyleSheet.create({ ... }); */
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flex: 1,
+  },
+  text: {
+    fontSize: 20,
+  },
+});
+/* @end */
 ```
 
+</SnackInline>
+
 ## API
 
 ```ts
 import * as Localization from 'expo-localization';
 ```
 
-## Constants
-
 ### Behavior
 
 This API is mostly synchronous and driven by constants. On iOS the constants will always be correct, on Android you should check if the locale has updated using `AppState` and `Localization.getLocalizationAsync()`. Initially the constants will be correct on both platforms, but on Android a user can change the language and return, more on this later.
 
+## Constants
+
 ### `Localization.locale`
 
-Native device language, returned in standard format. Ex: `en`, `en-US`, `es-US`.
+An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag), consisting of a 2 character language-code and optional script, region and variant codes. Ex: `en`, `en-US`, `zh-Hans`, `zh-Hans-CN`, `en-emodeng`.
 
 ### `Localization.locales`
 
 List of all the native languages provided by the user settings. These are returned in the order the user defines in their native settings.
 
 ### `Localization.region`
 
-**Available on iOS and Web.** Region code for your device which came from Region setting in Language & Region. Ex: US, NZ.
+Region code for your device which came from Region setting in Language & Region. This value is always available on iOS, but might return `null` on Android or web. Ex: `US`, `NZ`, `null`.
 
 ### `Localization.isoCurrencyCodes`
 
 A list of all the supported language ISO codes.
 
 ### `Localization.timezone`
 
-The current timezone in display format. ex: `America/Los_Angeles`
+The current timezone in display format. Ex: `America/Los_Angeles`.
 
 On Web `timezone` is calculated with `Intl.DateTimeFormat().resolvedOptions().timeZone`. For a better estimation you could use the `moment-timezone` package but it will add significant bloat to your website's bundle size.
 
 ### `Localization.isRTL`
 
 Returns if the system's language is written from Right-to-Left. This can be used to build features like [bidirectional icons](https://material.io/design/usability/bidirectionality.html).
 
+### `Localization.isMetric`
+
+Boolean value that indicates whether the system uses the metric system.
+
+### `Localization.currency`
+
+Three character ISO 4217 currency code. Not available on web. Ex: `USD`, `EUR`, `CNY`, `null`.
+
+### `Localization.decimalSeparator`
+
+Decimal separator used for formatting numbers. Ex: `,`, `.`.
+
+### `Localization.groupingSeparator`
```

Comment:
Make the name less ambiguous:

```suggestion
### `Localization.digitGroupingSeparator`
```

Reply:
Are you sure about this? Both iOS and Android use exactly this name, `groupingSeparator`, albeit within a subcontext of the localization thing. For instance, [react-native-localize](https://github.com/zoontek/react-native-localize) also uses the term `groupingSeparator`. Using `digitGroupingSeparator` would introduce a new term and `groupingSeparator` is quite often used within the localisation context in my experience.

Reply:
An alternative could be to put this in a sub-context like "numberFormat"

Reply:
The issue is that it is unclear what's being grouped. On iOS, groupingSeparator is a property of NumberFormatter. With Java, it's a property of DecimalFormatSymbols. "Thousands separator" would be clearer, too, although some locales do not group by thousands consistently - https://en.wikipedia.org/wiki/Indian_numbering_system.

Reply:
Not entirely, `groupingSeparator` is also a property of NSLocale on iOS.

This might be a matter of taste as well. I'd be good using `digitGroupingSeparator`, but we're introducing a new term. The term `groupingSeparator` is used universally and any searches within the iOS and Java docs will yield appropriate results.

Reply:
The main issue is that "grouping separator" in the context of localization alone isn't very precise. Is it used to group words or parts of a timestamp? If you search our docs for just "grouping separator" you'll still find `digitGroupingSeparator` or `numberGroupingSeparator` in the search results.

In short -- most APIs seem to include something about numbers in the name. Java uses `DecimalFormatSymbols.getGroupingSeparator()`, JS Intl uses the `Intl.NumberFormat` namespace, iOS has `NumberFormatter` and `NSLocale.groupingSeparator` as an exception, Microsoft calls this "digit grouping" and uses `NumberFormatInfo.NumberGroupSeparator` in .NET. Wikipedia also calls this "digit grouping".

Both `Localization.digitGroupingSeparator` and `Localization.Numbers.groupingSeparator` solve the issue. I don't think we are really introducing a new term since "digit grouping" and "number grouping" are used in established places, and if you are already thinking that "grouping separator" refers to numbers, "number/digit grouping separator" just reaffirms your thinking.

Reply:
Alright sounds good. I've updated it to `digitGroupingSeparator` 👍